src/Panel.vue

```diff
@@ -287,8 +287,9 @@
           this.$refs.retriever.fetch()
         }
       })
-      const id = this.$slots.header ? this.$refs.headerWrapper.innerText : jQuery(this.headerContent).text();
-      this.$refs.cardContainer.setAttribute("id", id.trim().replace(/\s+/g, '-').toLowerCase());
+      const panelHeader = this.$slots.header ? this.$refs.headerWrapper.innerHTML : this.headerContent;
+      const id = jQuery(panelHeader).wrap('<div></div>').parent().find(':header').text();
```

Comment:
`const panelHeaderText = ...` would be a more appropriate variable name (it is not an id at this stage).